src/main/java/se/sundsvall/seabloader/service/InvoiceService.java

```diff
@@ -29,4 +43,41 @@ public void create(final byte[] content) {
 
 		invoiceRepository.save(invoiceEntity);
 	}
+
+	public void exportInvoices() {
+		final var BATCH_SIZE = 10;
+		LOGGER.info("Exporting invoices to Invoice-cache");
+
+		final var invoiceIdsToSend = invoiceRepository.findInvoiceIdsByStatusIn(STATUSES_TO_SEND);
+
+		if (invoiceIdsToSend.isEmpty()) {
+			LOGGER.info("No invoices found with status {}", STATUSES_TO_SEND);
+			return;
+		}
+
+		LOGGER.info("Found {} invoices with status {}", invoiceIdsToSend.size(), STATUSES_TO_SEND);
+
+		new ArrayList<>(ListUtils
+			.partition(new ArrayList<>(invoiceIdsToSend), BATCH_SIZE))
+			.forEach(this::sendInvoiceToInvoiceCache);
+	}
+
+	private void sendInvoiceToInvoiceCache(final List<String> invoiceIds) {
+		invoiceIds.forEach(invoiceId ->
+			invoiceRepository.findByInvoiceId(invoiceId)
+				.ifPresent(invoiceEntity -> {
+					try {
+						final var invoicePdfRequest = toInvoicePdfRequest(toInExchangeInvoice(invoiceEntity.getContent()));
+						invoiceCacheClient.importInvoice(invoicePdfRequest);
+						invoiceEntity.setStatus(PROCESSED);
+						invoiceEntity.setContent(null);
```

Comment:
Why?

Reply:
We said last week that we could remove content after invoice is sent.

Reply:
Why?

Reply:
The space is already reserved. It will not help to set it to null, unless you also runs ` OPTIMIZE TABLE`.
I can't see that this is occuring. Hence: this is a useless action to perform.

Why not rely on the "database cleanup"-mechanizm instead?

Reply:
Fixed